### DISCUSSION TITLE
Zip Downloads

### DISCUSSION THREAD

**Discussion by PBhadoo** [spencerwooo/onedrive-vercel-index#507] [Q&A] [answered]
I don't know much about it, but are you using something to zip the folders on vercel or client side. Or the zip feature is from OneDrive itself.

Need this information for https://gitlab.com/ParveenBhadooOfficial/Google-Drive-Index

**Comment by spencerwooo** [comment, marked as answer]
It's done on the client side using jszip.

https://github.com/spencerwooo/onedrive-vercel-index/blob/7eb786025382146b6dbea478929c8e491e917594/components/MultiFileDownloader.tsx

And also cc @myl7 as he implemented the feature initially.
